### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SoongVilda**
Remove cacule

### CODE REVIEW

**Review by sirlucjan — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -16,13 +16,12 @@ We provided all of these schedulers because each scheduler performs differently
 - Task Type Scheduler by [Hamad Marri (TT)](https://github.com/hamadmarri/TT-CPU-Scheduler) - `linux-cachyos-tt` / `linux-tt`
 - BitMap Queue (BMQ) Alfred Chen Scheduler - `linux-cachyos-bmq`
 - Priority and Deadline based Skiplist multiple queue scheduler (PDS) - Alfred Chen Scheduler » `linux-cachyos-pds`
-- CacULE and CacULE-RDB created from Hamad Marri, maintained by CachyOS - `linux-cachyos-cacule` ATTENTION: Not supported after 6.1
```
Comment (ventureoo) [inline README.md:19]: I think we should not delete this line, but just underline that it is no longer supported

Comment (ptr1337) [reply README.md:19]: We can put a link to:
https://github.com/ptr1337/linux-cacule maybe.

Comment (ventureoo) [reply README.md:19]: * CacULE and CacULE-RDB created by Hamad Marri, supported by CachyOS in the past as - inux-cachyos-cacule
  **ATTENTION:** Not supported after version 6.1. If you still want to use it, you can get it from the archive repository - https://github.com/ptr1337/linux-cacule.

Comment (ventureoo) [reply README.md:19]: yep

Comment (ventureoo) [reply README.md:19]: so, let's just commit this?

**Pull request merged**